.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - run: sudo apt-get update
+      - run: sudo apt-get install ninja-build gcc-multilib
+      - run: pip3 install -U west
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          path: cobs
+
+      - uses: grandcentrix/actions-zephyr-sdk@v1
+        with:
+          url: https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.16.2-rc1/zephyr-sdk-0.16.2-rc1_linux-x86_64_minimal.tar.xz
+          toolchains: x86_64-zephyr-elf
+
+      - uses: grandcentrix/actions-ssh2https@v1
+        with:
+          token: ${{ secrets.REPO_READ_TOKEN }}
+
+      - name: Initialize
+        working-directory: cobs
+        run: |
+          west init -l --mf west-ci.yml .
+          west update -o--depth=1
+
+      - name: Install Python dependencies
+        run: pip3 install -r zephyr/scripts/requirements.txt
+
+      - name: Twister Tests
+        working-directory: cobs
+        run: |
+          west twister -vvvvv -T tests --integration
```

Comment:
I think -vv is enough. All important logs will be grabbed

Reply:
done